pkg/gateway/crds.go

```diff
@@ -29,6 +29,7 @@ import (
 
 // Ensure ensures that the required resources are deployed on the target system.
 // The resources handled here are the gateway CRDs: Cluster and Endpoint.
+//nolint:gocyclo // No further refactors necessary
```

Comment:
a space is needed here after // ?

Reply:
It complains about gcierror but running `gci write` on the file doesnot make any changes.

Reply:
`nolint` has to be introduced by `//` with no space, the explanatory comment needs a space. So this is fine as-is.